lib/mongo/mongo-schemaMap-generator.js

```diff
@@ -154,7 +166,7 @@ function generateForDefinitions(databaseName, definitions, dataKeyMap) {
     (accumulator, value) => {
       const collectionName = value.name
       const dataKey = dataKeyMap[collectionName]
-      accumulator[`${databaseName}.${collectionName}`] = generateForDefinition(value, dataKey)
+      accumulator[`${databaseName}.${collectionName}`] = generateForDefinition(value, dataKey, true)
```

Comment:
Is it possible to add a comment explaining that the third parameter set to true allows to start the recursion path in case the method on `fields` has to be called?